frontend/src/scenes/session-recordings/playlist/playlistLogic.ts

```diff
@@ -43,4 +43,18 @@ export const playlistLogic = kea<playlistLogicType>([
             }
         },
     })),
+    actionToUrl(({ values }) => ({
+        setIsFiltersExpanded: () => {
+            if (values.isFiltersExpanded === false) {
+                const newSearchParams = new URLSearchParams(router.values.currentLocation.search)
```

Comment:
I think you can use router.values.searchParams

that's already an object so you can just delete from it without needing to new-up a URLSearchParams